.github/stale.yml

```diff
@@ -0,0 +1,16 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 3
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 30
+# Pull requests with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
```

Comment:
we do not use these 2 labels , do we ?

we have priority labels used during the release (they should not get stale so not certain if it makes sense adding it here). I assume no labels for now here?